packages/react-core/src/components/Alert/Alert.js

```diff
@@ -35,7 +34,7 @@ const defaultProps = {
 
 const Alert = ({
   variant,
-  ariaLabel = `${capitalize(AlertVariant[variant])} Notification`,
+  'aria-label': ariaLabel = `${capitalize(AlertVariant[variant])} Notification`,
```

Comment:
It would be cleaner/clearer to assign arialLabel outside of the 'aria-label' assignment.

Reply:
I was trying to avoid using `delete props['aria-label']`; do you think we need to remove it from the spread props or cool to leave it in there? We are overwriting it on line 55/54 , but if the order of where we spread the props changes it is possible that could break. Unit tests would catch it though.

Reply:
I was just thinking:

```
const ariaLabel = `${capitalize(AlertVariant[variant])} Notification`;
const Alert = ({
  variant,
  'aria-label': ariaLabel,
```

Reply:
That unfortuniatly won't work since we need the variant from props. I could however make a function
```js
function getAriaLabel(variant) {
  return `${capitalize(AlertVariant[variant])} Notification`
}

const Alert = ({
  variant,
  'aria-label': ariaLabel = getAriaLabel(variant),
```